pom.xml

```diff
@@ -1559,6 +1559,7 @@
                       <allowedImport>org.rocksdb.TransactionLogIterator.BatchResult</allowedImport>
                       <allowedImport>org.rocksdb.TickerType</allowedImport>
                       <allowedImport>org.rocksdb.LiveFileMetaData</allowedImport>
+                      <allowedImport>org.rocksdb.BlockBasedTableConfig</allowedImport>
```

Comment:
Do we need this for some reason? If not, then good, if there is a place we use the BlockBasedTableConfig type outside the managed-rocksdb module, then we need to avoid that and use the ManagedBlockBasedTableConfig instead outside the managed rocksdb module.